grails-app/controllers/org/broadinstitute/orsp/SearchController.groovy

```diff
@@ -7,11 +7,12 @@ import org.grails.plugins.web.taglib.ApplicationTagLib
 import java.text.SimpleDateFormat
 
 @SuppressWarnings("GroovyAssignabilityCheck")
-class SearchController {
+class SearchController implements UserInfo {
```

Comment:
Really glad this pattern worked for you - I'd like to make more use of traits in this codebase.